game-backend/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "game-backend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "npm run serve",
+    "build": "npm run build-ts",
+    "serve": "node dist/server.js",
+    "test": "jest --forceExit --coverage --verbose",
```

Comment:
Maybe adding a pseudo unit test would be nice so we can start using the integration pipeline?
If we have no tests jest will always fail.

Reply:
Sure, I will make one.